### PR TITLE
Version 0.2.0 | Added functional basic RecyclerView and persistent data 

### LINKED ISSUE #2: Clean up Activities
These views have to be removed in future commits since we are using a RecyclerView

_Originally posted by @suptower in https://github.com/suptower/healthnote-android/pull/1#discussion_r875847040_

### PR DESCRIPTION
**Pull request by suptower**
The app now has all the functionality requested in the document for version 0.2.0, including a RecyclerView and persistent data.
The user is able to add medicaments to a list (only medicament name and ID are being stored right now, the latter is auto-generated)

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,12 +7,43 @@
     tools:context=".MainActivity">
 
     <TextView
```
Comment (suptower) [inline app/src/main/res/layout/activity_main.xml]: These views have to be removed in future commits since we are using a RecyclerView

**Comment by suptower** [comment]
Now closes #2 with 8f1fddd6c11043777599e562ac692d2b010a3a96

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (suptower) [reply app/src/main/res/layout/activity_main.xml]: Has been done in 8f1fddd6c11043777599e562ac692d2b010a3a96

**Review by HuberSimon — COMMENTED** [review]
Database review ok

**Review by binnaaa — APPROVED** [review]

**Pull request merged**